tests/std/tests/P0896R4_common_iterator_death/test.cpp

```diff
@@ -28,71 +28,158 @@ struct simple_input_iter {
+    (void) (cit.operator->()); // common_iterator can only be dereferenced if it holds an iterator
+}
+void test_case_operator_arrow_valueless() {
+    CIT cit{_Common_iterator_construct_tag{}};
+    (void) (cit.operator->()); // common_iterator can only be dereferenced if it holds an iterator
 }
 
-void test_case_operator_preincrement() {
+void test_case_operator_preincrement_sentinel() {
     CIT cit{default_sentinel};
-    ++cit; // cannot pre increment common_iterator that holds a sentinel
+    ++cit; // common_iterator can only be preincremented if it holds an iterator
 }
 
-void test_case_operator_increment() {
+void test_case_operator_preincrement_valueless() {
+    CIT cit{_Common_iterator_construct_tag{}};
+    ++cit; // common_iterator can only be preincremented if it holds an iterator
+}
+
+void test_case_operator_postincrement_sentinel() {
     CIT cit{default_sentinel};
-    cit++; // cannot increment common_iterator that holds a sentinel
+    cit++; // common_iterator can only be postreincremented if it holds an iterator
+}
+
+void test_case_operator_postincrement_valueless() {
+    CIT cit{_Common_iterator_construct_tag{}};
+    cit++; // common_iterator can only be postincremented if it holds an iterator
 }
 
-void test_case_iter_move() {
+void test_case_equality_left_valueless() {
+    CIT cit1{_Common_iterator_construct_tag{}};
+    CIT cit2{};
+    (void) (cit1 == cit2); // common_iterator can only be compared if it holds a value
+}
+
+void test_case_equality_right_valueless() {
+    CIT cit1{};
+    CIT cit2{_Common_iterator_construct_tag{}};
+    (void) (cit1 == cit2); // common_iterator can only be compared if it holds a value
+}
+
+void test_case_difference_left_valueless() {
+    CIT cit1{_Common_iterator_construct_tag{}};
+    CIT cit2{};
+    (void) (cit1 - cit2); // common_iterator can only be subtracted if it holds a value
+}
+
+void test_case_difference_right_valueless() {
+    CIT cit1{};
+    CIT cit2{_Common_iterator_construct_tag{}};
+    (void) (cit1 - cit2); // common_iterator can only be subtracted if it holds a value
+}
+
+void test_case_iter_move_sentinel() {
     CIT cit{default_sentinel};
-    (void) ranges::iter_move(cit); // cannot iter_move common_iterator that holds a sentinel
+    (void) ranges::iter_move(cit); // common_iterator can only be iter_moved if it holds an interator
 }
 
-void test_case_iter_swap_sentinel_left() {
+void test_case_iter_move_valueless() {
+    CIT cit{_Common_iterator_construct_tag{}};
+    (void) ranges::iter_move(cit); // common_iterator can only be iter_moved if it holds an interator
+}
+
+void test_case_iter_swap_sentinel_left_sentinel() {
     CIT cit1{default_sentinel};
     CIT cit2{};
-    (void) ranges::iter_swap(cit1, cit2); // cannot iter_swap common_iterators that hold a sentinel
+    (void) ranges::iter_swap(cit1, cit2); // common_iterator can only be iter_swapped if it holds an interator

```

Comment:
The actual message is "Can only iter_swap common_iterators if both hold an iterator". Ditto inconsistently capitalized, ditto should match. (Also on 143 and 149.)